server/services/zwave/lib/comClass/switchMultilevelCommandClass.js

```diff
@@ -0,0 +1,119 @@
+const { DefaultCommandClass } = require('./defaultCommandClass');
+const logger = require('../../../../utils/logger');
+const { COMMAND_CLASSES, INDEXES } = require('../constants');
+
+const SwitchMultilevelCommandClass = function SwitchMultilevelCommandClass() {
+  DefaultCommandClass.call(this);
```

Comment:
Same here, I don't think we want this kind of inheritance here, it complexifies the code for not much value added. 

Can you just use functions? The simpler, the better

Reply:
See previous response, the goal is to get a real object definition here to represent CommandClasses. All CommandClasses have some generic behavior and some specific ones. POO is perfectly adapted for this.